.semaphore/semaphore.yml

```diff
@@ -294,3 +294,13 @@ blocks:
         - name: python/starlette
           commands:
             - "rake app=python/starlette app:test"
+  - name: Run tests for PHP test setups
+    dependencies: ["Build processmon"]
+    task:
+      prologue:
+        commands:
+          - "echo APPSIGNAL_PUSH_API_KEY=not-a-real-api-key > appsignal_key.env"
+      jobs:
+        - name: php/laravel-collector
+          commands:
+            - "rake app=php/laravel-collector app:test"
```

Comment:
I don't know why, but this fails. It works locally, but on CI it keeps saying "The app has not started yet. Retrying..."
Meanwhile, the app is reporting data, and I can see the collector receive traces from those requests.

https://appsignal.semaphoreci.com/jobs/c7f0e05e-3a64-442b-8456-16dac3330f2f/plain_logs.txt